pkgs/development/libraries/gobject-introspection/default.nix

```diff
@@ -59,19 +59,22 @@ stdenv.mkDerivation rec {
     meson
     ninja
     pkg-config
+    flex
+    bison
     gtk-doc
     docbook-xsl-nons
-    docbook_xml_dtd_43
+    docbook_xml_dtd_43 # FIXME: remove in next release
     docbook_xml_dtd_45
+    python3
+    setupHook # move .gir files
   ];
 
   buildInputs = [
-    flex
-    bison
     python3
```

Comment:
cc @FRidh from a cross-compiling perspective we usually used the build-native python interpreter to build python packages, no?

Reply:
yes, the build-native one is used in `buildPythonPackage`.

Reply:
maybe they link against the interpreter?

Reply:
Build scripts need build python and some gobject utils are written in python so they need host python.